### PR TITLE
0xcb 1337 - Added keymap "conor" and fixed some minor stuff 

### PR DESCRIPTION
**Pull request by Conor-Burns**
## Description
- added keymap "conor"
- fixed RGB led number
- changed screen layout around
- updated readme

## Types of Changes

- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/0xcb/1337/config.h [hunk 1]
```diff
@@ -47,7 +47,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define RGB_DI_PIN D3
 #ifdef RGB_DI_PIN
-  #define RGBLED_NUM 5
+  #define RGBLED_NUM 4
```
Comment (drashna) [inline keyboards/0xcb/1337/config.h:50]: This shouldn't be changed here.  This should be changed in your keymaps config.h.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/0xcb/1337/config.h (hunk 1 above)
Comment (Conor-Burns) [reply keyboards/0xcb/1337/config.h:50]: But the pcb only has 4 rgb LEDs why is it keymap specific?

Comment (Conor-Burns) [reply keyboards/0xcb/1337/config.h:50]: I put 5 in there by mistake

**Review by drashna — APPROVED** [review]

File: keyboards/0xcb/1337/1337.c [hunk 2]
```diff
@@ -24,9 +24,11 @@ void eeconfig_init_kb(void) {
 #ifdef RGBLIGHT_ENABLE
     rgblight_enable();
     rgblight_sethsv(0, 255, 255);
-#ifdef RGBLIGHT_ANIMATIONS
+// clang-format off
+    #ifdef RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/0xcb/1337/1337.c:28]:
```suggestion
#    ifdef RGBLIGHT_ANIMATIONS
```

File: keyboards/0xcb/1337/1337.c [hunk 3]
```diff
@@ -24,9 +24,11 @@ void eeconfig_init_kb(void) {
...
     rgblight_mode(RGBLIGHT_MODE_RAINBOW_SWIRL + 2);
-#endif
+    #endif
```
Comment (drashna) [inline keyboards/0xcb/1337/1337.c:30]:
```suggestion
#    endif
```

File: keyboards/0xcb/1337/keymaps/conor/keymap.c [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    oled_write_P(led_state.num_lock ? PSTR("NUM ") : PSTR("    "), false);
+    oled_write_P(led_state.caps_lock ? PSTR("CAP ") : PSTR("    "), false);
+    oled_write_ln_P(led_state.scroll_lock ? PSTR("SCR ") : PSTR("    "), false);
+}
+static void render_rgbled_status(bool full) {
+#ifdef RGBLIGHT_ENABLE
+  char buf[30];
+  if (RGBLIGHT_MODES > 1 && rgblight_is_enabled() && get_highest_layer(layer_state) == _RGB) {
+      if (full) {
+          snprintf(buf, sizeof(buf), "RGB mode %2d: %d,%d,%d  \n",
```
Comment (drashna) [inline keyboards/0xcb/1337/keymaps/conor/keymap.c:141]: Just a heads up, snprintf can use 1.5k of firmware space.  Instead, breaking it up into sections and then just rendering numbers like this:
```c
    char string[4];
    uint16_t n = get_value();
    string[3] = '\0';
    string[2] = '0' + n % 10;
    string[1] = ( n /= 10) % 10 ? '0' + (n) % 10 : (n / 10) % 10 ? '0' : ' ';
    string[0] =  n / 10 ? '0' + n / 10 : ' ';
    oled_write(string, false);
```

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by Conor-Burns** [comment]
@drashna @fauxpark Thank you!

**Pull request merged**